src/rpc/RPCHelpers.cpp

```diff
@@ -1350,6 +1350,19 @@ parseTaker(boost::json::value const& taker)
         return Status{RippledError::rpcBAD_ISSUER, "invalidTakerAccount"};
     return *takerID;
 }
+
+ripple::Issue
+parseIssue(boost::json::object const& issue)
+{
+    Json::Value jv;
+    if (issue.contains(JS(issuer)))
+        jv["issuer"] = issue.at(JS(issuer)).as_string().c_str();
```

Comment:
The issuer should be an account, which will be validated. 
Since we have #953 , How about we leave this to a dedicated task to fix them in one place?

Reply:
I was thinking it is better to note create more work for the future. But maybe it would be solved by creating a new function in utils so maybe it is actually better to be solved all at once. So maybe it is fine to leave it as it is if you don't want to change it.